package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "workflow-dispatch-and-wait",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

Comment:
Most actions have seen have treated going from Node 16 to Node 20 as a breaking change.